### ISSUE TITLE
Scaling input for deep learning

### ISSUE THREAD

**Issue by hammad93** [hammad93/hurricane-net#1]
Previously, the deep learning model has an exploding gradient because the features were not scaled and the tensors converged to NaN.

After further analysis, the best way to standardize and scale the features using RobustScaler (http://scikit-learn.org/0.18/modules/generated/sklearn.preprocessing.RobustScaler.html#sklearn.preprocessing.RobustScaler) 

`For the scaler to work correctly, we need to find a way to remove NaN.`

**Comment by hammad93** [comment]
The minimum pressure is parsed as None or NaN because of the older data. We will continue with the model with only complete data.
